src/deploymentstatus.h

```diff
@@ -10,33 +10,30 @@
 
 #include <limits>
 
-/** Global cache for versionbits deployment status */
-extern VersionBitsCache g_versionbitscache;
-
 /** Determine if a deployment is active for the next block */
-inline bool DeploymentActiveAfter(const CBlockIndex* pindexPrev, const Consensus::Params& params, Consensus::BuriedDeployment dep)
+inline bool DeploymentActiveAfter(const CBlockIndex* pindexPrev, const Consensus::Params& params, Consensus::BuriedDeployment dep, [[maybe_unused]] VersionBitsCache& versionbitscache)
```

Comment:
versionbitscache is not used by DepdloymentActiveAfter for burried forks - hm; strange that bitcoin did it like that.
it complicates code so much

Reply:
Resolved in latest push!